src/libraries/System.Net.Quic/src/System/Net/Quic/Interop/msquic_generated_macos.cs

```diff
@@ -111,6 +111,9 @@ internal static unsafe partial class MsQuic_MacOS
         [NativeTypeName("#define QUIC_STATUS_CERT_NO_CERT QUIC_STATUS_CERT_ERROR(3)")]
         public const int QUIC_STATUS_CERT_NO_CERT = ((int)(3) + 512 + 200000000);
 
+        [NativeTypeName("#define QUIC_STATUS_ADDRESS_NOT_AVAILABLE ((QUIC_STATUS)EADDRNOTAVAIL)")]
+        public const int QUIC_STATUS_ADDRESS_NOT_AVAILABLE = ((int)(0x63));
```

Comment:
Are you sure the code is 0x63 on MacOS? Most of the codes have different values on MacOs and this StackOverflow post seems to suggest it is different https://stackoverflow.com/questions/2244409/eaddrnotavail-when-binding-127-0-0-1-on-localhost

Reply:
you're right, I'll fix this also on msquic